conferences/migrations/0005_auto_20161217_1238.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
In python 3 you don't need to define coding: http://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python3